### ISSUE TITLE
(#132) Maintain the same order on the Receipts and Acknowledgements

### ISSUE THREAD

**Issue by brprashanth** [rxtrax/tracking#101]
1. Request you to maintain the same order on the Receipts in which the items are added on to the list.
2. The Hippaa Acknowledgements as well should be displayed in the same order as it is being requested on the IPAD.

**Comment by brprashanth** [comment]
The item order is still not fixed. The acknowledgement order is fixed.

**Comment by jjamminjim** [comment]
@brprashanth when you say `item` order, you mean transaction items?

**Comment by brprashanth** [comment]
@jjamminjim  No, this is about delivery-prescriptions. We discussed about this in one of our calls. 
We send itemized delivery prescriptions. Item A Rx 1, Item B Rx 2, etc etc in a particular order.
We want the same order to be displayed on the receipts.

**Comment by jjamminjim** [comment]
Can you create a new delivery that shows the issue? I believe that existing deliveries may not show the fix. @jasonmadrid is that correct?

**Comment by jasonmadrid** [comment]
@brprashanth @jjamminjim correct, this will only be work for new deliveries created since the new build.

**Comment by jjamminjim** [comment]
@brprashanth Jason and I just test and it seems to be working.

**Comment by brprashanth** [comment]
Thanks Jim this seem to be working now. Will reopen if required.

**Comment by archanamckesson** [comment]
Trivial issue - It seems to appear again.

**Comment by brprashanth** [comment]
This issue is occuring again. The items on the receipt does not maintain the same order.
